app/components/learnergroup-members.js

```diff
@@ -122,38 +134,106 @@ export default Ember.Component.extend({
   }.property('members.[]', 'topLevelGroup', 'topLevelGroup.allDescendants.@each'),
-  actions: {
-    changeLearnerGroup: function(groupIdString, userId){
-      this.set('saving', true);
-      let groupId = parseInt(groupIdString);
-      let toSave = [];
-      this.get('store').find('user', userId).then(
-        user => {
-          toSave.pushObject(user);
-          this.get('topLevelGroup').then(topLevelGroup => {
-            topLevelGroup.removeUserFromGroupAndAllDescendants(user).then(groups=>{
-              toSave.pushObjects(groups);
-              if(groupId === -1){
-                //we're moving this user out of the group into the cohort
-                //so just save
-                Ember.RSVP.all(toSave.uniq().invoke('save')).then(()=>{
-                  this.set('saving', false);
-                });
-              } else {
-                this.get('store').find('learnerGroup', groupId).then( learnerGroup => {
-                  learnerGroup.addUserToGroupAndAllParents(user).then(groups =>{
-                    toSave.pushObjects(groups);
-                    Ember.RSVP.all(toSave.uniq().invoke('save')).then(()=>{
-                      this.set('saving', false);
-                    });
+
+  sendPutRequests(groupIdString, userId) {
+    this.set('saving', true);
+    let groupId = parseInt(groupIdString);
+    let toSave = [];
+    let component = this;
+
+    this.get('store').find('user', userId).then(
+      (user) => {
+        toSave.pushObject(user);
+        component.get('topLevelGroup').then((topLevelGroup) => {
+          topLevelGroup.removeUserFromGroupAndAllDescendants(user).then((groups) => {
+            toSave.pushObjects(groups);
+            if (groupId === -1) {
+              // we're moving this user out of the group into the cohort
+              // so just save
+              all(toSave.uniq().invoke('save')).finally(() => {
+                component.set('saving', false);
+              });
+            } else {
+              component.get('store').find('learnerGroup', groupId).then(learnerGroup => {
+                learnerGroup.addUserToGroupAndAllParents(user).then(groups =>{
+                  toSave.pushObjects(groups);
+                  all(toSave.uniq().invoke('save')).finally(() => {
+                    component.set('saving', false);
                   });
                 });
-              }
-              
-            });
+              });
+            }
           });
-        }
-      );
+        });
+      }
+    );
+  },
+
+  actions: {
+    changeLearnerGroup(groupIdString, userId){
+      this.sendPutRequests(groupIdString, userId);
+    },
+
+    addStudent(studentId) {
+      this.get('toBulkSave').pushObject(studentId);
+    },
+
+    removeStudent(studentId) {
+      this.get('toBulkSave').removeObject(studentId);
+    },
+
+    continueButton() {
+      if (isEmpty(this.get('toBulkSave'))) {
+        this.set('errorMessage', 'You must checkmark at least one.');
```

Comment:
Instead of having an error and a string that has to be translated, how about if the button is just disabled when nothing has been checked.  I think thats clear enough.